avocado_vt/plugins/vt_cluster.py

```
@@ -0,0 +1,248 @@
+            """Helper function to cleanup agent on a single node."""
+            node_dir = os.path.join(cluster_dir, node.name)
+            os.makedirs(node_dir, exist_ok=True)
+
+            upload_error = None
+            stop_error = None
+
+            try:
+                remote_path = node.proxy.core.get_agent_log_filename()
+                if remote_path:
+                    node.copy_files_from(node_dir, remote_path)
+            except Exception as err:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)